pkg/termination/write.go

```diff
@@ -41,6 +48,11 @@ func WriteMessage(path string, pro []v1alpha1.PipelineResourceResult) error {
 	if err != nil {
 		return err
 	}
+
+	if len(jsonOutput) > MaxContainerTerminationMessageLength {
+		return MessageLengthError(fmt.Sprintf("Size of container termination message is above max size allowed: %d, cause by large task result.", MaxContainerTerminationMessageLength))
```

Comment:
How about: `Termination message is above max allowed size (%d), caused by large task result.`

Reply:
Done